src/kernel/src/net/mod.rs

```diff
@@ -51,7 +52,11 @@ impl NetManager {
             MessageFlags::from_bits_retain(flags)
         };
 
-        todo!()
+        let msg = unsafe { &mut *msg };
+
+        info!("Recieving a message {msg:?} from fd {fd} with flags {flags:?}.");
+
+        Ok((msg.iovec_len as i32).into())
```

Comment:
This basically a stub because it response with unexpected data. Some games might handle this correctly but some games may not, which may cause segmentation fault. In the later case it can cause headache for us to find what it the cause of segmentation fault.

Reply:
It is a stub, sorry. I accidentally left it in here.